libr/core/panels.c

```diff
@@ -5233,7 +5236,7 @@ static void __add_menu(RCore *core, const char *parent, const char *name, RPanel
 	}
 	item->n_sub = 0;
 	item->selectedIndex = 0;
-	item->name = name ? r_str_new (name) : NULL;
+	item->name = name ? R_STR_DUP (name) : NULL;
```

Comment:
```suggestion
	item->name = R_STR_DUP (name);
```

same for this no need to chk for null twice